js/externals.js

```diff
@@ -1,94 +1,139 @@
+      'components/GridField/GridFieldRow': 'GridFieldRow',
+      'components/GridField/GridFieldTable': 'GridFieldTable',
+      'components/HiddenField/HiddenField': 'HiddenField',
+      'components/ListGroup/ListGroup': 'ListGroup',
+      'components/ListGroup/ListGroupItem': 'ListGroupItem',
+      'components/LiteralField/LiteralField': 'LiteralField',
+      'components/Loading/Loading': 'Loading',
+      'components/PopoverField/PopoverField': 'PopoverField',
+      'components/Preview/Preview': 'Preview',
+      'components/ResizeAware/ResizeAware': 'ResizeAware',
+      'components/Search/Search': 'Search',
+      'components/Search/SearchToggle': 'SearchToggle',
+      'components/Tag/CompactTagList': 'CompactTagList',
+      'components/Tag/Tag': 'Tag',
+      'components/Tag/TagList': 'TagList',
+      'components/TextField/TextField': 'TextField',
+      'components/Tip/Tip': 'Tip',
+      'components/Toolbar/Toolbar': 'Toolbar',
+      'components/TreeDropdownField/TreeDropdownField': 'TreeDropdownField',
+      'components/TreeDropdownField/TreeDropdownFieldMenu': 'TreeDropdownFieldMenu',
+      'components/TreeDropdownField/TreeDropdownFieldNode': 'TreeDropdownFieldNode',
+      'components/VersionedBadge/VersionedBadge': 'VersionedBadge',
+      'components/ViewModeToggle/ViewModeToggle': 'ViewModeToggle',
+      'containers/FormBuilderLoader/FormBuilderLoader': 'FormBuilderLoader',
+      'containers/InsertLinkModal/InsertLinkModal': 'InsertLinkModal',
+      'containers/InsertLinkModal/fileSchemaModalHandler': 'FileSchemaModalHandler',
+      'lib/Backend': 'Backend',
+      'lib/Config': 'Config',
+      'lib/DataFormat': 'DataFormat',
+      'lib/formatWrittenNumber': 'formatWrittenNumber',
+      'lib/getFormState': 'getFormState',
+      'lib/Injector': 'Injector',
+      'lib/ReactRouteRegister': 'ReactRouteRegister',
+      'lib/reduxFieldReducer': 'reduxFieldReducer',
+      'lib/Router': 'Router',
+      'lib/schemaFieldValues': 'schemaFieldValues',
+      'lib/ShortcodeSerialiser': 'ShortcodeSerialiser',
+      'lib/SilverStripeComponent': 'SilverStripeComponent',
+      'lib/TinyMCEActionRegistrar': 'TinyMCEActionRegistrar',
+      'lib/withDragDropContext': 'withDragDropContext',
+      'lib/withRouter': 'withRouter',
+      'state/breadcrumbs/BreadcrumbsActions': 'BreadcrumbsActions',
+      'state/records/RecordsActions': 'RecordsActions',
+      'state/records/RecordsActionTypes': 'RecordsActionTypes',
+      'state/schema/SchemaActions': 'SchemaActions',
+      'state/tabs/TabsActions': 'TabsActions',
+      'state/toasts/ToastsActions': 'ToastsActions',
+      'state/unsavedForms/UnsavedFormsActions': 'UnsavedFormsActions',
+      'state/viewMode/ViewModeActions': 'ViewModeActions',
+      'state/viewMode/ViewModeStates': 'ViewModeStates',
+      // bundles/bundle.js aliases
+      config: 'Config', // alias for lib/Config
+      // bundles/vendor.js
+      '@apollo/client': 'ApolloClient',
```

Comment:
```suggestion
      // This can't be exposed - see https://github.com/webpack-contrib/expose-loader/issues/188
      // '@apollo/client': 'ApolloClient',
```